client/components/listingsView.jsx

```diff
@@ -1,33 +1,176 @@
+  cursor: pointer;
+`;
+
+const NextBtn = styled.div`
+  display: ${props => props.counter === -3072? 'none' : ''};
+  position: absolute;
+  top: 25%;
+  z-index: 10;
+  right: 7.4%;
+  font-size: 30px;
+  color: white;
+  cursor: pointer;
+`;
+
+const Heart = styled.div`
+  stroke: white;
+  stroke-width: 50;
+  opacity: .7;
+  color: #565656;
+  position: absolute;
+  top: 4%;
+  z-index: 10;
+  right: 18%;
+  font-size: 30px;
+  cursor: pointer;
+`;
+
+class ListingsView extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      counter: 0,
+    }
+    this.increaseCounter = this.increaseCounter.bind(this);
+    this.descreaseCounter = this.decreaseCounter.bind(this);
+  }
+
+  increaseCounter() {
+    if (this.state.counter >= 0) {
+      return;
+    }
+    this.setState({
+      counter: this.state.counter + 1024,
+    })
+  }
+
+  decreaseCounter() {
+    if (this.state.counter <= -3072) {
+      return;
+    }
+    this.setState({
+      counter: this.state.counter - 1024,
+    })
+  }
 
    render() {
      return (
-      <span display="flex">{this.props.listings.map((listing) =>
-        <Fragment>
-          <img src={listing.imageUrl}></img>
-          <h3>{listing.price}</h3>
-          <h3>{listing.bedrooms}</h3>
-          <h3>{listing.bathrooms}</h3>
-          <h3>{listing.sqft}</h3>
-          <h3>{listing.address}</h3>
-        </Fragment>
-          )}
-      </span>
+        <div>
+        {/* <Wrapper> */}
+          <PrevBtn counter={this.state.counter}>
+            <FontAwesomeIcon onClick={() => {this.increaseCounter()}} icon={faChevronCircleLeft}></FontAwesomeIcon>
+          </PrevBtn>
+          <Main className="mainContainer">{this.props.listings.map((listing) =>
+            <Fragment>
+              <ListingContainer counter={this.state.counter}>
+                <ImageContainer>
+                  <Image src={listing.imageUrl}></Image>
+                </ImageContainer>
+                  <Price>{listing.price}</Price>
+                <SpecsStyle>
+                  <StyledIcon icon={faBed}/>
+                  <Text>{listing.bedrooms}</Text>
+                  <StyledIcon icon={faBath}/>
+                  <Text>{listing.bathrooms}</Text>
+                  <StyledIcon icon={faRulerCombined}/>
+                  <Text>{listing.sqft}</Text>
+                  <Heart>
+                    <FontAwesomeIcon icon={faHeart}/>
+                  </Heart>
+                </SpecsStyle>
+                <SpecsStyle><Text>{listing.address}</Text></SpecsStyle>
+                <SpecsStyle><Text>{'Waialae-Kahala, Honolulu, HI'}</Text></SpecsStyle>
+              </ListingContainer>
+            </Fragment>
+              )}
+          </Main>
+          <NextBtn counter={this.state.counter}>
+            <FontAwesomeIcon onClick={() => {this.descreaseCounter()}} icon={faChevronCircleRight}></FontAwesomeIcon>
+          </NextBtn>
+        {/* </Wrapper> */}
+        </div>
```

Comment:
This is super clean @tonyperletti! Love your use of Styled Components here in this render. Very well done :)